pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -235,6 +235,9 @@ const (
 	// for the request. This data is synthetic; it is not stored in etcd and instead is only applied when retrieving
 	// CRs for the CRD.
 	AnnotationAPIIdentityKey = "apis.kcp.dev/identity"
+	// AnnotationAPIExportExtraKeyPrefix is the prefix of an annotation set on an APIExport to provide extra info. The
+	// annotation with the prefix will be synced to the all the APIBindings bound to this APIExport.
+	AnnotationAPIExportExtraKeyPrefix = "extra.apis.kcp.dev/"
```

Comment:
If by default this annotation is expected to be added to `APIExport`s and then replicated automatically to bound `APIBinding`s, why defining it in the go file related to  `APIBinding`s ? I would have expected it would be defined in `types_apiexport.go`, but maybe there's a reason I'm missing ?

@sttts I assume such an annotation was what you had initially in mind. WDYT about the syntax used here:
```
extra.apis.kcp.dev/<annotation name>
```
In our [preliminary design session](https://docs.google.com/document/d/10Tloa3IZ36PV5JJodxKeA3p65ivVqgpwmqqsOwUWewg/edit#bookmark=id.erq3116xk1sh) , the proposed syntax was `apis.kcp.dev/extra-<annotationName>`, but I tend to prefer adding `extra` in the annotation key prefix segment, and let the key name segment be available for the annotation name defined by third parties. And since the name segment of an annotation key supports dots:

> The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between

that allows scoping the annotation name with the package of the third party:
```
extra.apis.kcp.dev/compute.workload.kcp.dev
```

Reply:
Which annotation ends up on the binding? Without the extra prefix?